run.bash

```diff
@@ -53,14 +67,16 @@ fi
 
 # Run Lava.
 
-config=${LAVA_CONFIG:-"${GITHUB_ACTION_PATH}/default.yaml"}
 output=$(mktemp)
 
-lava scan -c "${config}" > "${output}"
+lava scan -c "${config}" >> "${output}"
```

Comment:
I would keep `>` here to make explicit that this is the first (and only) write to `${output}`.

```suggestion
lava scan -c "${config}" > "${output}"
```